src/app/components/countriesCard/Card.tsx

```diff
@@ -15,13 +15,13 @@ export default function CountryCard(props: CountryCardProps) {
     <section>
       <AppContainer>
         <div className="w-80  h-custom-height shadow-lg rounded-xl bg-white cursor-pointer">
-        <div className="relative h-40 w-full shadow-md">
+        <div className="relative h-48 w-full shadow-md">
             <Image
               src={props.flag}
               alt={props.alt}
               layout="fill"
               objectFit="cover"
-              className="absolute inset-0 rounded-t-xl"
+              className="inset-0 rounded-t-xl object-contain"
```

Comment:
Instead of adding object-contain, I propose that you just change the objectFit attribute to "contain" on line 23